prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html

```diff
@@ -108,6 +108,7 @@
                               startView="multi-year"
                               [touchUi]="isMobile"></mat-datepicker>
               <mat-error *ngIf="renewalDate.hasError('required')">Required</mat-error>
+              <mat-error *ngIf="renewalDate.hasError('future')">Must be in the future</mat-error>
```

Comment:
"Renewal date must be in the future"?

Reply:
hm.. because the error message is right under the control, I don't think we need to repeat the field name. Just like "Required" instead of "Renewal Date is required", trying to follow the same theme.